sqpnp/sqpnp.cpp

```diff
@@ -289,13 +342,12 @@ namespace sqpnp
     H(3, 4) = r[6] - dot_j5q2*H(3, 1) - dot_j5q4*H(3, 3); H(4, 4) = r[7] - dot_j5q2*H(4, 1) - dot_j5q4*H(4, 3); H(5, 4) = r[8] - dot_j5q2*H(5, 1) - dot_j5q4*H(5, 3);
     H(6, 4) = r[3] - dot_j5q3*H(6, 2); H(7, 4) = r[4] - dot_j5q3*H(7, 2); H(8, 4) = r[5] - dot_j5q3*H(8, 2);
     
-    H.block<9, 1>(0, 4) /= H.col(4).norm();
+    H.block<9, 1>(0, 4) *= (1.0 / H.col(4).norm());
```

Comment:
No need for parenthesis.

Reply:
True but I find the code more readable with them.